index.html

```diff
@@ -5,180 +5,183 @@
-            </p>
-            <button type="button"><span>Get my CV</span><span><img class="arrow-download" src="images/ic_download.svg" alt="arrow download"></span></button>
-            <div id="Languages">
-                <h3>Languages</h3>
-              <ul>
-                  <li>JavaScript(ES5/ES6)</li>
-                  <li>Ruby</li>
-                  <li>HTML</li>
-                  <li>CSS</li>
-              </ul>
-            </div>
-  
-            <div id="Frameworks">
-                <h3>Frameworks</h3>
-              <ul>
-                  <li>Bootstrap</li>
-                  <li>Ruby on rails</li>
-                  <li>RSpec</li>
-                  <li>Capibara</li>
-                  <li>Selenium</li>
-              </ul>
-            </div>
-  
-          <div id="Skills">
-              <h3>Skills</h3>
-            <ul>
-                <li>Codekit</li>
-                <li>Codepen</li>
-                <li>GitHub</li>
-                <li>GitLAb</li>
-                <li>Terminal</li>
-            </ul>
-          </div>
-        </section>  
-        <!--Get Started Section-->
-      <section id="get-started">
-        <img class="get-started-img" src="images/image_geometry_1.svg" alt="image_geometry_3">
-        <div class="container">
-          <h2>Get started</h2>
-          <p class="get-started-content">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
-        </p>
-        <form action="https://formspree.io/f/mlezegyq" method="POST">
-            <input type="text" id="name" name="user_name" maxlength="30" placeholder="Your name" required/>
-            <input type="email" id="email" name="user_email" placeholder="Your email" required/>
-            <textarea id="msg" name="user_message" maxlength="500" placeholder="Enter your message here..." required></textarea>
-            <button class="submit-form" type="submit">Start collaboration</button>
-        </form>
-        <div class="social-media">
-            <h3><span>Get my Resume</span><span><img class="img-resume" src="images/ic_link_teal.svg" alt="img-resume"></span></h3>
-             <ul class="social-media-list">
-                <li><a href=""><img src="images/ic_stackoverflow.svg" alt="stackoverflow"></a></li>
-                <li><a href="https://github.com/ForHemer"><img src="images/ic_github.svg" alt="GitHub"></a></li>
-                <li><a href="http://twitter.com/HemersonForeste"><img src="images/ic_twitter.svg" alt="twitter"></a></li>
-                <li><a href="https://www.linkedin.com/in/hemerson-foreste-890685197/"><img src="images/ic_linkedin.svg" alt="Linkedin"></a></li>
-                <li><a href=""><img src="images/ic_medium.svg" alt="Medium"></a></li>
-                <li><a href=""><img src="images/ic_angellist.svg" alt="Angellist"></a></li>
-            </ul>
-        </div>
-        </div>
-      </section>
```

Comment:
- [x]  The same issue in this part the form section is not showing up as you can see in the screenshots below, Kindly, fix this issue🙏🏼☺️.

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/149597171-fb9471d0-0ee0-4657-a828-e9ac78268d4e.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/149597174-5e2d09c8-e754-440b-b8f6-be50e9dea616.PNG)